site/zenodo_rdm/legacy/serializers/schemas.py

```diff
@@ -145,11 +202,11 @@ def hook_imprint_partof(self, data, **kwargs):
         publisher = data.get("publisher")
         if imprint:
             data["part_of"] = imprint
-        if publisher:
-            data["custom_fields"]["imprint:imprint"] = {
-                **imprint,
-                "publisher": publisher,
-            }
+            if publisher:
```

Comment:
is this indent intended? (tongue twister haha)

Reply:
yes, the RDM custom field `imprint:imprint` serializes to two fields in legacy (`imprint` and `part_of`). 

If only `imprint` is set, we can hook it to `part_of` (serializes to `partof_title` and `partof_pages`).

On the other hand if there's also a publisher, we add it to the imprint information (that serializes to `imprint_publisher`, `imprint_isbn` and `imprint_place`).